crates/host/src/module/wasmer_sys.rs

```diff
@@ -49,18 +49,21 @@ pub fn make_runtime_engine() -> Engine {
     Engine::headless()
 }
 
-/// Take WASM binary and prepare a wasmer Module suitable for iOS
-pub fn build_ios_module(wasm: &[u8]) -> Result<Module, CompileError> {
-    info!(
-        "Found wasm and was instructed to serialize it for ios in wasmer format, doing so now..."
-    );
-    let compiler_engine = make_engine();
-    let store = Store::new(compiler_engine);
-    Module::from_binary(&store, wasm)
-}
+pub struct PreCompiledSerializedModule {}
```

Comment:
Remove references to iOS, in function name and comments, as this functionality may be useful in other platforms as well